code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Toxins.dm

```diff
@@ -645,16 +648,45 @@
 	description = "Spectrocybin is a hallucinogenic chemical found in a unique strain of fungi. Little research has been conducted into the hallucinogenic properties of spectrocybin, though many spiritual creeds utilise the drug in rituals and claim it allows people to act as mediums between the living and dead."
 	reagent_state = LIQUID
 	color = "#800080"
-	strength = 5
+	strength = 5 // Was pretty strong during testing - liver failure before you hit the spooky messages.
+	overdose = 10
 	taste_description = "acid"
-	metabolism = REM
+	metabolism = REM * 0.5 // Reduced from 0.2u/t to 0.1u/t to prolong the progression from good messages to bad.
 	unaffected_species = IS_DIONA | IS_MACHINE
+	var/datum/modifier/modifier
 
 /datum/reagent/toxin/spectrocybin/affect_blood(var/mob/living/carbon/M, var/removed)
 	..()
-	M.hallucination = max(M.hallucination, 50)
+	if(is_overdosed == FALSE)	
+		M.hallucination = max(M.hallucination, 20) //Lowered from 50 to 20 due to the new flavour messages that will be appearing in chat ontop of hallucination messages.
+		if(prob(20)) //Increased from 10% chance to 20% chance just to make it more likely for a ghost to be spotted.
+			M.see_invisible = SEE_INVISIBLE_CULT
+	if(dose < 5)
+		if(prob(10))
+			M.emote("shiver")
```

Comment:
Not sure this works, should use the appropriate decl.

Reply:
Resolved in a brief discussion. 👍